### PR TITLE
Mandate SVG logo in docs

### PR DESCRIPTION
**Pull request by djptek**
Add guidance on image format to mandate `svg` for logos since `png` may crash Kibana. `png` is fine for other images, this only applies to the integration logo.

## What does this PR do?

Updated the generic guidelines to mandate use of `svg` images for Integration logo after observing that `png` crashes the Integrations browser

## Checklist

*this is a change to docs, not a new integration*

- [x] ~I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.~
- [x] ~I have verified that all data streams collect metrics or logs.~
- [x] ~I have added an entry to my package's `changelog.yml` file.~
- [x] ~I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).~

### CODE REVIEW

File: docs/generic_guidelines.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Integration packages should provide out-of-the-box dashboards.
 
 #### Content for elastic.co/integrations
 
-Each integration will be listed on the public website elastic.co/integrations and the package registry will serve as the source of truth. As a result, our docs and screenshots should be high quality to showcase the integration. Any additional branding material should be reviewed, e.g.:
+Each integration will be listed on the public website elastic.co/integrations and the package registry will serve as the source of truth. As a result, our docs and screenshots should be high quality to showcase the integration. Please ensure to use `svg` for logs and `png` for all other images. Any additional branding material should be reviewed, e.g.:
```
Comment (mtojek) [inline docs/generic_guidelines.md:95]: nit: logos?

**Comment by andrewkroh** [comment]
If we only support svg then perhaps we should enforce this in the package-spec. Like require that the file name have a `.svg` extension or add an enum to the `type` field.

https://github.com/elastic/package-spec/blob/cfbd7e7bf2124f5c2f1a3f4119036b5d14bf6de8/versions/1/integration/manifest.spec.yml#L79

**Review by mtojek — APPROVED** [review]

**Review by andrewkroh — APPROVED** [review]

**Comment by jen-huang** [comment]
> > If we only support svg then perhaps we should enforce this in the package-spec. Like require that the file name have a .svg extension or add an enum to the type field.
> 
> We should examine existing packages if this is right for all packages. @jen-huang any comments about this idea?

The restriction to svg only for logos sounds like a good idea to me. Fleet renders the logos using [EuiIcon](https://elastic.github.io/eui/#/display/icons) under the hood, which only supports SVGs (and React elements, but that's not applicable here): https://github.com/elastic/kibana/blob/8880884c9193cf209661d5e037e768b190d69869/x-pack/plugins/fleet/public/components/package_icon.tsx

**Comment by mtojek** [comment]
This is just about docs. Let's merge it.

**Pull request merged**